sdk/tests/conformance2/rendering/element-index-uint.html

```diff
@@ -265,7 +264,7 @@
     shouldBeUndefined('gl.drawElements(gl.TRIANGLES, 3, gl.UNSIGNED_INT, 0)');
     wtu.glErrorShouldBe(gl, gl.NO_ERROR);
     debug("Enable normals, out-of-range");
-    gl.vertexAttribPointer(normalLoc, 3, gl.FLOAT, false, 7 * sizeInBytes(gl.FLOAT), 4 * sizeInBytes(gl.FLOAT));
+    gl.vertexAttribPointer(normalLoc, 3, gl.FLOAT, false, 7 * sizeInBytes(gl.FLOAT), 21 * sizeInBytes(gl.FLOAT));
```

Comment:
@kenrussell Thanks for your suggestion. I have followed sdk/tests/conformance/rendering/out-of-bounds-index-buffers.html to rewrite the out of range test case.  Currently, I move all out of range related tests to function runIndexOutOfRangeTests.  runCopiesIndicesTests was also merged into this function.